f2f-ipv-stub/template.yaml

```diff
@@ -296,9 +298,12 @@ Resources:
         Variables:
           SIGNING_KEY: !Ref IPVStubKMSSigningKey
           CLIENT_ID: !FindInMap [Configuration, !Ref Environment, IPVStubID]
-          REDIRECT_URI: !Sub "https://ipvstub.review-c.${Environment}.account.gov.uk/redirect"
+          REDIRECT_URI: !Sub
+            - "https://${AWS::StackName}-${IPVSTUBURL}/redirect"
```

Comment:
Do we think we'll have multiple stubs in Build env to or could this be conditional just for dev?

Reply:
modified to allow multiple stubs in Dev env only